_tuning-your-cluster/availability-and-recovery/remote.md

```diff
@@ -228,8 +230,5 @@ You can use remote-backed storage for the following purposes:
 
 The following are known limitations of the remote-backed storage feature:
 
-- Writing data to a remote store can be a high-latency operation when compared to writing data on the local file system. This may impact the indexing throughput and latency.
-- Primary-to-primary relocation is unstable because handover of translog uploads from older to new primary has not been implemented.
-- Garbage collection of the metadata file has not been implemented.
-For other limitations, see the [Remote store known issue list](https://github.com/opensearch-project/OpenSearch/issues/5678).
+- Writing data to a remote store can be a high-latency operation when compared to writing data on the local file system. This may impact the indexing throughput and latency. Performance benchamrking results will be tracked in https://github.com/opensearch-project/OpenSearch/issues/6376
```

Comment:
```suggestion
- Writing data to a remote store can be a high-latency operation when compared to writing data on the local file system. This may impact the indexing throughput and latency. For performance benchmarking results, see [issue #6376](https://github.com/opensearch-project/OpenSearch/issues/6376).
```